### PR TITLE
feat: add showAttachedCropToolbar option to CropToolbarConfig

### PR DESCRIPTION
**Pull request by guoyingtao**
This PR solves #173 

@vanniktech 
I added showBottomCropToolbar option to CropToolbarConfig, you can check the example `Hide Rotation Dial`

image
image

### CODE REVIEW

File: Mantis.podspec [hunk 1]
```diff
@@ -21,7 +21,7 @@ Pod::Spec.new do |s|
   s.social_media_url   = "http://twitter.com/guoyingtao"
   s.platform     = :ios
   s.swift_version = "5.0"
-  s.ios.deployment_target = "11.0"
```
Comment (vanniktech) [inline Mantis.podspec:24]: Is this change really necessary? I still have `deployment_target = "12.0"` for my app

Comment (guoyingtao) [reply Mantis.podspec:24]: It is because of my previous PR #169 
I added a CropToolbarIconProvider also I decided to use SF Symbol for CropToolbar icons instead of drawing them by myself which needs iOS 13
With apple's iOS 16 coming soon, I think it might be a good timing to move to iOS 13 because it also supports Combine, SwiftUI etc.

**Comment by guoyingtao** [comment]
@vanniktech 
I decided to just support iOS 11.0+ again and use `if #available` when using `SF symbols` for now

**Review by vanniktech — APPROVED** [review]
Lovely, thanks!
